osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/dlmsobjectconfig/DlmsObjectConfigSmr51.java

```diff
@@ -47,6 +51,15 @@
 
     objectList.addAll(Arrays.asList(gsmDiagnosticGprs, gsmDiagnosticCdma, gsmDiagnosticLteM));
 
+    // Additional clear mbus status for SMR5.1
+    final DlmsObject readMBusStatus =
+        new DlmsObject(READ_MBUS_STATUS, InterfaceClass.EXTENDED_REGISTER.id(), "0.<c>.24.2.6.255");
+    final DlmsObject clearMBusStatus =
+        new DlmsObject(CLEAR_MBUS_STATUS, InterfaceClass.DATA.id(), "0.<c>.94.31.10.255");
+    final DlmsObject clientSetupMBus =
+        new DlmsObject(CLIENT_SETUP_MBUS, InterfaceClass.MBUS_CLIENT.id(), "0.<c>.24.1.0.255");
+    objectList.addAll(Arrays.asList(readMBusStatus, clearMBusStatus, clientSetupMBus));
+
```

Comment:
These objects are now all added to the SMR5.1 config. 
However:
- object clientSetupMBus (0.<c>.24.1.0.255) is also specified in the DSMR422 and SMR5
- object readMBusStatus (0.<c>.24.2.6.255) is also specified in SMR5 (but not in DSMR422)
Only the clearMBusStatus object is new in SMR5.1.

So you could consider adding the clientSetupMBus and readMBusStatus objects to the other configs as well. It is not necessary for this story, but it could prevent confusion later on.

Reply:
The objects were added to the other config objects